src/app/onion/dashboard/components/splash/splash.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="splash">
+  <div class="splash__inner">
+    <div class="splash__top">
+      <button (click)="navigateBack.emit()" class="splash__back">
+        <i class="far fa-arrow-left"></i> Back
+      </button>
+      <button class="splash__user">
```

Comment:
Aria-label please. Also, since we are using the user profile piece in two different places could we make it it's own component to make accessibility a little easier.

Reply:
We can definitely make that its own component, but it might be a good idea to do it in a separate PR to avoid bloating this one with more components.